common/persistence/cassandraPersistence.go

```diff
@@ -136,6 +136,7 @@ const (
 		`create_request_id: ?, ` +
 		`decision_version: ?, ` +
 		`decision_schedule_id: ?, ` +
+		`decision_schedule_time: ?, ` +
```

Comment:
Probably we should just have a visibility_ts on transfer task for we have the same mechanism for each kind of task for delaying them.